pkg/controller/utils/discovery.go

```diff
@@ -132,3 +135,14 @@ func isDockerEE(c *kubernetes.Clientset) (bool, error) {
 	}
 	return false, nil
 }
+
+// isEKS returns true if running on an EKS cluster, and false otherwise.
+// EKS doesn't have any provider-specific API groups, so we need to use a different approach than
+// we use for other platforms in autodetectFromGroup.
+func isEKS(c kubernetes.Interface) (bool, error) {
+	cm, err := c.CoreV1().ConfigMaps("kube-system").Get("eks-certificates-controller", metav1.GetOptions{})
```

Comment:
Presumably we're already shipping with RBAC that allows this check, right?

Reply:
I hadn't specifically checked before but did now. The operator has a ClusterRole with all verbs allowed for configmaps. Thanks for asking.